### PR TITLE
RANCHER-1894: Create new pipeline updateSidecarFromFeatureBranch

### PR DESCRIPTION
**Pull request by yaroslavishchenko**
Related Jira Ticket:
[RANCHER-1894: Create new pipeline updateSidecarFromFeatureBranch](https://folio-org.atlassian.net/browse/RANCHER-1894)

### CODE REVIEW

File: pipelines/folioRancher/folioDevTools/moduleDeployment/deploySidecarFromFeatureBranch/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,157 @@
+package folioRancher.folioDevTools.moduleDeployment.deploySidecarFromFeatureBranch
+
+import org.folio.Constants
+import org.folio.models.EurekaNamespace
+import org.folio.utilities.Logger
+import org.folio.utilities.GitHubClient
+import org.jenkinsci.plugins.workflow.libs.Library
+
+@Library('pipelines-shared-library@RANCHER-1894') _
```
Comment (OHaimanov) [inline pipelines/folioRancher/folioDevTools/moduleDeployment/deploySidecarFromFeatureBranch/Jenkinsfile:9]: Please do not forget to set correct branch before merge

File: pipelines/folioRancher/folioDevTools/moduleDeployment/deploySidecarFromFeatureBranch/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+/** Pipeline properties and parameters */
+properties([
+  buildDiscarder(logRotator(numToKeepStr: '30')),
+  disableConcurrentBuilds(),
+  parameters([
+    string(name: 'MODULE_NAME', defaultValue: 'folio-module-sidecar', description: 'GitHub Repository Name for Sidecar application'),
```
Comment (OHaimanov) [inline pipelines/folioRancher/folioDevTools/moduleDeployment/deploySidecarFromFeatureBranch/Jenkinsfile:16]: If it is espesially job for sidecar only, we could remove this parameter

File: pipelines/folioRancher/folioDevTools/moduleDeployment/deploySidecarFromFeatureBranch/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    folioParameters.branchWithRef('MODULE_BRANCH', 'MODULE_NAME'),
```
Comment (OHaimanov) [inline pipelines/folioRancher/folioDevTools/moduleDeployment/deploySidecarFromFeatureBranch/Jenkinsfile:17]: And refer here by expilcit name

File: pipelines/folioRancher/folioDevTools/moduleDeployment/deploySidecarFromFeatureBranch/Jenkinsfile (hunk 2 above)
Comment (yaroslavishchenko) [reply pipelines/folioRancher/folioDevTools/moduleDeployment/deploySidecarFromFeatureBranch/Jenkinsfile:16]: @OHaimanov 

What if Sidecar name is changed some day or we switch to another one?

**Review by eldiiar-duishenaliev — APPROVED** [review]
That's really good, all suggestions are implemented.

**Pull request merged**